docs/pages/ops-guide/overview.md

```diff
@@ -310,27 +310,37 @@ The path of a secret defines which service IDs can have access to it. You can th
 
 ### Binary Secrets
 
-You can store binary files, like a Kerberos keytab, in the DC/OS secrets store. Your file must be Base64-encoded as specified in RFC 4648.
+You can store binary files, like a Kerberos keytab, in the DC/OS secrets store. In DC/OS 1.11+ you can create secrets from binary files directly, while in DC/OS 1.10 or lower, files must be base64-encoded as specified in RFC 4648 prior to being stored as secrets.
+
+#### DC/OS 1.11+
+
+To create a secret called `mysecret` with the binary contents of `kerb5.keytab` run:
 
-You can use standard `base64` command line utility. The following example uses the BSD `base64` command.
 ```
-$  base64 -i krb5.keytab -o kerb5.keytab.base64-encoded
+$ dcos security secrets create --file kerb5.keytab mysecret
 ```
 
-The `base64` command line utility in Linux inserts line-feeds in the encoded data by default. Disable line-wrapping with the `-w 0` argument. Here is a sample base64 command in Linux.
+#### DC/OS 1.10 or lower
+
+To create a secret called `mysecret` with the binary contents of `kerb5.keytab`, first encode it using the `base64` command line utility. The following example uses BSD `base64` (default on macOS).
+
 ```
-$  base64 -w 0 -i krb5.keytab > kerb5.keytab.base64-encoded
+$ base64 -i krb5.keytab -o kerb5.keytab.base64-encoded
```

Comment:
We could specify the long options `--input` and `--output` instead.

Also `bash` tag.